spinnaker-monitoring-daemon/spinnaker-monitoring/__main__.py

```diff
@@ -171,28 +170,21 @@ def set_default_paths():
   abs_path = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
   registry_path = os.path.join(abs_path, 'registry')
   dev_path = os.path.join(abs_path, 'registry.dev')
-  yml_path = os.path.join(abs_path, 'spinnaker-monitoring.yml')
 
   if os.path.exists(registry_path):
     spectator_client.DEFAULT_REGISTRY_DIR = registry_path
   elif os.path.exists(dev_path):
     spectator_client.DEFAULT_REGISTRY_DIR = dev_path
 
-  home_yml_path = os.path.join(
-      os.environ.get('HOME', ''), '.spinnaker', 'spinnaker-monitoring.yml')
-  hal_yml_path = os.path.join(
-      os.environ.get('HOME', ''),
-      '.hal', 'default', 'profiles', 'spinnaker-monitoring.yml')
-
-  global DEFAULT_CONFIG_PATH
-  if os.path.exists(home_yml_path):
-    DEFAULT_CONFIG_PATH = home_yml_path
-  elif os.path.exists(hal_yml_path):
-    DEFAULT_CONFIG_PATH = hal_yml_path
-  elif os.path.exists(yml_path):
-    DEFAULT_CONFIG_PATH = yml_path
+  home_dir = os.environ.get('HOME', '')
+  return [
+      '/opt/spinnaker-monitoring/config',
+      os.path.join(abs_path),
+      os.path.join(home_dir, '.hal', 'default', 'profiles'),
+      os.path.join(home_dir, '.spinnaker')
+  ]
 
 
 if __name__ == '__main__':
-  set_default_paths()
-  main()
+  search_path = set_default_paths()
```

Comment:
Is there a side effect in `set_default_paths()`? Otherwise I think this can be one line.

Reply:
it could be, but doesnt seem to help anything.
It's only a two line function to begin with.